tests/test_trainor_collection.py

```diff
@@ -128,4 +128,4 @@ def test_ephys_training(tmp_path):
 
     # Validation is a bit random due to initilization. We check that you get
```

Comment:
Silly question: but there's no way to pass a random number generator around so that you can control the seed that is fed to the optimizer (I assume there is an optimizer somewhere that is the source of the randomness)?

Reply:
That is an interesting point. It might go beyond the goal of this PR.  Will not address this but we can look into this with a later PR.